README.md

```diff
@@ -9,4 +9,42 @@ The `.browserslistrc` file is a list of browsers that front-end tools will targe
 You can test your Browserslist query configurations via [this tool](http://browserl.ist/).
```

Comment:
I would like to see a little more introduction up on line 2 about the different ways someone could use this repo. Something like

>This repo is intended to be used as either a template for new webpack projects or as a resource for webpack and other related configs.

Doesn't have to be long, just a way of framing the project and giving people an idea of what to expect.

Reply:
I added this. Wasn't sure if you wanted the first sentence removed or not. Let me know.